aspnetcore/includes/net-core-prereqs-windows.md

```diff
@@ -1,6 +1,6 @@
 [Visual Studio for Windows](https://www.microsoft.com/net/download/windows).  
```

Comment:
Visual Studio for Windows --> Visual Studio 2017 with the following workloads: (use a bulleted list below for the 2 workload names)